src/browser/transport/mod.rs

```diff
@@ -84,7 +84,7 @@ impl Transport {
 
         let open = Arc::new(AtomicBool::new(true));
 
-        let (shutdown_tx, shutdown_rx) = mpsc::channel();
+        let (shutdown_tx, shutdown_rx) = mpsc::sync_channel(1);
```

Comment:
Can you take a larger buffer value here ? like 50 or 100, a small buffer size can impact peformance

Reply:
Fixed.